### DISCUSSION TITLE
Create a space in Context menu strip or center text in Item menu

### DISCUSSION THREAD

**Discussion by omid99visual** [dotnet/csharplang#4900] [Q&A] [answered]
Hi, I had a question ... how can I set the distance between items in **Windows form** and using the tool?
Or align the text property of the menu so that the text below the icon is in the middle?
.
Thank you for answering ...

**Comment by theunrepentantgeek** [comment, marked as answer]
This repo is for discussions about the design and evolution of the C# language itself, which makes it the wrong place to ask questions about Windows Forms. I'd suggest you try asking on [StackOverflow](https://stackoverflow.com/), though you may find that this level of customization requires using a custom control suite.

**Reply by omid99visual** [reply]
Thank you very much, I will definitely ask my questions in **Stack Overflow.**

**Reply by jnm2** [reply]
@omid99visual Another place that is less formal than Stack Overflow is https://discord.gg/csharp (listed in https://github.com/dotnet/csharplang/blob/main/Communities.md) which has a `#gui` channel where Windows Forms how-to questions would be on topic.

**Reply by omid99visual** [reply]
> @omid99visual Another place that is less formal than Stack Overflow is https://discord.gg/csharp (listed in https://github.com/dotnet/csharplang/blob/main/Communities.md) which has a `#gui` channel where Windows Forms how-to questions would be on topic.

Thanks a lot
